design/ceph/object/store.md

```diff
@@ -119,3 +119,39 @@ Specifying this section also ensures that the pool section in the ceph-object-zo
   zone:
     name: "name"
 ```
+
+### LDAP
+
+The Ceph Object Gateway supports integrating with LDAP for authenticating and creating users, please refer [here](https://docs.ceph.com/docs/master/radosgw/ldap-auth/). This means that the `ceph object store user` is also required to be part of groups in the LDAP server otherwise, authentication will fail. The following settings need to be configured in the RGW server:
+```
+rgw ldap binddn =
+rgw_ldap_secret = /etc/ceph/ldap/bindpass.secret
+rgw ldap uri =
+rgw ldap searchdn =
+rgw ldap dnattr =
+rgw ldap searchfilter =
+rgw_s3_auth_use_ldap = true
```

Comment:
Inconsistencies in underscores here. Why is that?

Reply:
These are names used by `radosgw`: 

https://docs.ceph.com/en/quincy/radosgw/ldap-auth/#configuring-the-ceph-object-gateway-to-use-ldap-authentication

Reply:
oh, you mean ` ` vs `_`. Yes, I agree. The names should match the literal config values.